### PR TITLE
Kubeyaml multi arch image

### LINKED ISSUE #56: arm64 flux kubeyaml issue
Hi
looks like the kubeyaml image referenced in flux dockerfile can not run on arm64
https://github.com/raspbernetes/multi-arch-images/blob/78b3faaac4a36fc3e80619da15d1608bdf7616f8/build/flux/Dockerfile#L46

after i applied a helmrelease which should be automated by flux i get the following issues within the flux log.
```
gitops/flux-c55549f45-2jct2[flux]: ts=2020-03-14T23:02:45.336631844Z caller=loop.go:151 component=sync-loop jobID=465e0132-22fe-9a78-4aa9-91285f059732 state=done success=false err="applying changes: updating resource monitoring:helmrelease/infra-grafana in fluxcd/monitoring/grafana/infra-grafana_hr.yaml: fork/exec /usr/lib/kubeyaml/kubeyaml: exec format error"
```
and flux is unable to upgrade the helmrelease to a newer image.

Can you check included image?

Thanks in advance

### PR DESCRIPTION
**Pull request by HaveFun83**
Hi guys,

thanks for your awesome work here. I had some time today to work on the kubeyaml issue.

This PR should fix #56 but first test this build here and than change the image ref within the flux build.
Please have a look feedback is welcome.

### CODE REVIEW

**Comment by HaveFun83** [comment]
hmm maybe someone can check this login issue?

**Comment by xUnholy** [comment]
Looks like you've done a great job, I'll take a look into the failure.

If you could also update the `.github/workflows/schedule.yml` file with the step to check for any upstream changes, this is how we track and maintain parity with upstream releases that would be fantastic!

**Comment by xUnholy** [comment]
I've just figured out why the login fails because this is running in a branch it is actually looking for these credentials configured in your own repository and hence they wouldn't exist - ignore the failure in this case, and once you've updated the PR with the schedule task update we should be ready to merge!

File: build/kubeyaml/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+WORKDIR /tmp
+RUN wget https://github.com/squaremo/kubeyaml/archive/${VERSION}.tar.gz \
+  && tar xvf ${VERSION}.tar.gz  \
+  && mkdir -p /home/kubeyaml \
+  && cp /tmp/kubeyaml-${VERSION}/kubeyaml.py /home/kubeyaml/ \
+  && cp /tmp/kubeyaml-${VERSION}/requirements.txt /home/kubeyaml/
+
+WORKDIR /home/kubeyaml
+
+#COPY kubeyaml.py requirements.txt ./
```
Comment (xUnholy) [inline build/kubeyaml/Dockerfile]: If the commented lines aren't required we can safely remove them.

Comment (HaveFun83) [reply build/kubeyaml/Dockerfile]: removed from dockerfile

**Comment by HaveFun83** [comment]
> Looks like you've done a great job, I'll take a look into the failure.
> 
> If you could also update the `.github/workflows/schedule.yml` file with the step to check for any upstream changes, this is how we track and maintain parity with upstream releases that would be fantastic!

I added kubeyaml to the schedule.yml

**Pull request merged**